ansible/deploy_proxy.yml

```diff
@@ -9,26 +9,20 @@
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.dhis == True, action: site, site_name: dhis_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech1 == True, action: site, site_name: motech1 }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech1 == True, action: site, site_name: motech1_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech2 == True, action: site, site_name: motech2 }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech2 == True, action: site, site_name: motech2_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.wiki == True, action: site, site_name: wiki }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.wiki == True, action: site, site_name: wiki_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech}
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.riakcs == True, action: site, site_name: riakcs }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: cchq_redirect }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: cchq_http }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: cchq_http_redirect }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: hq_status }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: dhis }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: motech1 }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: motech2 }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: wiki }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: motech}
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: riakcs }
     # Don't put any .commcarehq.org ssl configurations below commtrack, the ssl cert variable that gets loaded persists between roles
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.commtrack_ssl == True, action: site, site_name: commtrack_ssl }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.commtrack_http == True, action: site, site_name: commtrack_http }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.icds_tableau == True, action: site, site_name: icds_tableau }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.icds_tableau == True, action: site, site_name: icds_tableau_http }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: commtrack_ssl }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: commtrack_http }
+    - { role: nginx, when: proxy_type == 'nginx', action: site, site_name: icds_tableau }
```

Comment:
I like this change, if for no other reason than the lines are shorter and easier to read.